bower.json

```diff
@@ -1,7 +1,8 @@
 {
   "name": "bedrock-credential-curator",
   "main": [
-    "./components/credential-curator.js"
+    "./components/credential-curator.js",
+    "./components/credential-task/credential-task.less"
```

Comment:
We have `components/main.less` in other projects.  Does naming apply to sub-component dirs too or could that get confusing with multiple `main.less`es?  `component.less`?  I'll leave it as is for now as this is an easy change later.